packages/react-components/src/index.css

```diff
@@ -1,11 +1,17 @@
 :root {
-    --o-ui-focus-ring: 0 0 0 2px var(--white), 0 0 0 4px var(--primary-500);
     --o-ui-disabled-opacity: 0.4;
     --o-ui-pill-border-radius: 999px;
     --o-ui-rounded-border-radius: var(--scale-alpha);
     --o-ui-circular-border-radius: 100%;
 }
 
+[class^="o-ui-"] {
```

Comment:
I guess you added this for specifity? If so, shouldn't we add all the vars here?

Reply:
for specificity as well to be able to use custom properties in custom properties. All of our component specific custom properties could be indeed prefixed.

Reply:
yes they should be!